code/jose/Python_Labs/lab02.py

```diff
@@ -5,47 +5,46 @@
 # hundreds_digit = x//100
 # print(hundreds_digit)
 
+random_numbers = {20: 'twenty', 30: 'thirty', 40: 'forty', 50: 'fifty', 60: 'sixty',70: 'seventy',80: 'eighty',90: 'ninety',100: 'one hundred',200: 'two hundred',300: 'three hundred',
+400: 'four hundred',500: 'five hundred',600: 'six hundred',700: 'seven hundred',800: 'eight hundred',900: 'nine hundred',999 : 'nine hundred ninety nine'}
 
-ones = {1: 'one',
-2: 'two',
-3: 'three',
-4: 'four',
-5: 'five',
-6: 'six',
-7: 'seven',
-8: 'eight',
-9: 'nine'
-}
-tens_under20 = {10: 'ten',
-11: 'eleven',
-12: 'twelve',
-13: 'thirteen',
-14: 'fourteen',
-15: 'fifteen',
-16: 'sixteen',
-17: 'seventeen',
-18: 'eighteen',
-19: 'nineteen'
-}
-tens_digitwritten = {2: 'twenty',
-3: 'thirty',
-4: 'forty',
-5: 'fifty',
-6: 'sixty',
-7: 'seventy',
-8: 'eighty',
-9: 'ninety'}
+ones = {0 : 'zero',1: 'one',2: 'two',3: 'three',4: 'four',5: 'five',6: 'six',7: 'seven',8: 'eight',9: 'nine'}
```

Comment:
This is not critical to grading, but it can make the code more readable to have the dictionaries spread over lines like you had in the previeus version:
Better formatting:
```
tens_digitwritten = {2: 'twenty',
3: 'thirty',
4: 'forty',
5: 'fifty',
```

Reply:
makes sense. I pulled them closer so the code was not so long down but ill keep them as before